avocado/utils/build.py

```diff
@@ -18,6 +18,36 @@
 from . import process
 
 
+def configure(path, configure=None):
+    """
+    Configures the source tree for a subsequent build
+
+    Most source directories coming from official released tarballs
+    will have a "configure" script, but source code snapshots may have
+    "autogen.sh" instead (which usually creates and runs a "configure"
+    script itself).  This function will attempt to run the first one
+    found (if a configure script name not given explicitly).
+
+    :param configure: the name of the configure script (None for trying to
+                      find one automatically)
+    :type configure: str or None
+    :returns: the configure script exit status, or None if no script was
+              found and executed
+    """
+    cwd = os.getcwd()
+    try:
+        os.chdir(path)
+        if configure is not None:
+            return process.run(os.path.join(path, configure)).exit_status
```

Comment:
Hmm, do we want to do the "os.path.join"? I'd leave it on the user which way to go as they provide the path. "./configure" should do the trick but without the join users could configure into the expected dst dir while specifying the configure from a different directory.

Reply:
IIRC there was a reason for doing the join... I'll check that and either change it or not on v2.

Reply:
sure, I'd prefer that but I don't insist on it...

Reply:
So I've verified, and we either need to add `./` to the configure candidates, or the path, which IMO, is more precise and should give more information in case of errors.

Reply:
That is expected, anyway I'm fine with keeping this as is until some user complains...